cbibs_api/api.py

```diff
@@ -131,8 +136,48 @@ def get(self):
 class RetrieveCurrentReadings(BaseResource):
     keys = ['constellation', 'station']
     method_decorators = [check_api_key_and_req_type]
+    def __init__(self):
+        sql_name = self.__class__.__name__
+        self.constellation = request.args.get('constellation', 'cbibs')
+        self.station = request.args.get('station')
+        now = datetime.utcnow()
+        start_date = now - relativedelta(weeks=2)
+        params = {'constellation':self.constellation, 'station':self.station, 'start_date':start_date}
+        self.table = pd.read_sql(SQL[sql_name], db.engine, params=params)
```

Comment:
Could also do `pd.read_sql(SQL[sql_name], db.engine, params=params).dropna(axis=1, subset='obs_value')` to avoid side effects.